include/zephyr/net/wifi_mgmt.h

```diff
@@ -145,6 +145,7 @@ enum net_event_wifi_cmd {
 	NET_EVENT_WIFI_CMD_RAW_SCAN_RESULT,
 	/** Disconnect complete */
 	NET_EVENT_WIFI_CMD_DISCONNECT_COMPLETE,
+	NET_EVENT_WIFI_CMD_CONNECT_STATUS,
```

Comment:
This should get a doxygen docstring, as other commands.

Reply:
Done.